relay/config.go

```diff
@@ -0,0 +1,44 @@
+package relay
+
+import core "github.com/Layr-Labs/eigenda/core/v2"
+
+// Config is the configuration for the relay Server.
+type Config struct {
+
+	// Shards contains the IDs of the relays that this server is willing to serve data for. If empty, the server will
+	// serve data for any shard it can.
+	Shards []core.RelayKey
```

Comment:
"contains the IDs of the relays", then shouldn't it be called `relays`? Unclear what is shard here

Reply:
I've been using the term shard and relay ID interchangeably... will update the naming here to use the term "relay ID"